docs/reference/query-dsl/match-query.asciidoc

```diff
@@ -142,21 +142,6 @@ out of the analyzed text. For example:
 }
 --------------------------------------------------
 
-Since `match_phrase` is only a `type` of a `match` query, it can also be
-used in the following manner:
-
-[source,js]
---------------------------------------------------
-{
-    "match" : {
-        "message" : {
-            "query" : "this is a test",
-            "type" : "phrase"
-        }
-    }
-}
---------------------------------------------------
-
```

Comment:
MY personal opinion is that we should only document the current way of doing things and support the deprecated syntax only for people who have already written their code to give them a chance to update their code during the current major version (in this case 5.x). That way we don't accidentally encourage people to use the old syntax and get them into trouble for upgrading to the next major version (in this case 6.0). 

That's just my thoughts though. Happy to see what @clintongormley thinks :)